eclipse-collections-api/src/main/java/org/eclipse/collections/api/bag/sorted/ImmutableSortedBag.java

```diff
@@ -137,6 +137,15 @@
     @Override
     ImmutableSortedBag<T> dropWhile(Predicate<? super T> predicate);
 
+    /**
+     * @since 8.2
+     */
+    @Override
+    default <V> ImmutableSortedBag<V> countBy(Function<? super T, ? extends V> function)
+    {
+        return this.asLazy().<V>collect(function).toSortedBag().toImmutable();
```

Comment:
I would have to go with an unsorted bag today, as we do not have an ordered bag (yet).